README.md

```
@@ -139,4 +139,7 @@ Having trouble or facing issues with Mycroft GUI ? You can use the below guide t
    
    - Run `mycroft-enclosure-client` This will start the client and show Debug() messages on the console.
 
+### Fullscreen Mode for ovos-gui-app 
```

Markdownlint

Expected: setext; Actual: atx
Heading style

(MD003, heading-style)